ansible/all-in-one/roles/xsnippet/tasks/main.yml

```diff
@@ -148,3 +202,10 @@
     enabled: yes
     state: restarted
     scope: user
+
+- name: proxy HTTP and HTTPS from host to containers
```

Comment:
Good catch! I'd thought that all variables are shared between roles, but that's not the case.